CMakeLists.txt

```diff
@@ -115,6 +115,10 @@ if(UNIX)
     set(CPACK_PACKAGING_INSTALL_PREFIX "/opt/rocm/hip" CACHE PATH "HIP Package Installation Path")
```

Comment:
I'd add the following two lines here:
```cmake
set(HIPBINDIR ${CPACK_PACKAGING_INSTALL_PREFIX}/bin)
set(ROCMBINDIR ${CPACK_PACKAGING_INSTALL_PREFIX}/../bin)
```

And remove the following two lines in the below `packaging/hipify-clang.*` files:
```cmake
HIPBINDIR="/opt/rocm/hip/bin"
ROCMBINDIR="/opt/rocm/bin"
```

Could you test it on all the target OSs, please?
Thank you!

Reply:
Tested it out. The post install files don't have visibility of the local variables in CMakeLists.txt.

Reply:
Aha. I thought that not only `hipify-clang.txt` does have visibility of local variables declared in `CMakeLists.txt`.